ani-cli

```diff
@@ -116,16 +116,13 @@ dep_ch() {
 }
 
 where_iina() {
-    [ -n "$ANI_CLI_PLAYER" ] && printf "%s" "$ANI_CLI_PLAYER" && return 0
-    command -v iina >/dev/null && printf "iina" && return 0
     [ -e "/Applications/IINA.app/Contents/MacOS/iina-cli" ] && echo "/Applications/IINA.app/Contents/MacOS/iina-cli" && return 0
-    dep_ch iina # exit with formating
+    printf "%s" "iina" && return 0
```

Comment:
Do we need that last one?

Reply:
yes, as i said on discord, the uname case is only for assumption . this will be `dep_ch`ed  after the args so the user knows that the default is iina if they havent installed it.